static/js/modules/filters-event.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+
+var FilterPanel = require('fec-style/js/filter-panel').FilterPanel;
+var URI = require('urijs');
+
+function lineNumberFilters() {
+  this.filterPanel = new FilterPanel();
```

Comment:
I tried to just have the listener on `$('#filters form')` but it doesn't seem to work. I'm guessing it's because `FilterSet()` activates all the various form handlers for events to be attached to?